tests/reference_data/methane.json

```diff
@@ -1,11 +1,11 @@
 {"1": {
- "cell": [[0.0, 0.0, 0.0], [0.0, 0.0, 0.0], [0.0, 0.0, 0.0]],
- "ctime": 18.923226180085212,
- "mtime": 18.923226180085212,
+ "cell": [[20.0, 0.0, 0.0], [0.0, 20.0, 0.0], [0.0, 0.0, 20.0]],
+ "ctime": 19.826526159217163,
```

Comment:
changes here are annoying, as well as in the unique_id & user fields. Anyway to leave these out?

Reply:
No, the previous molecule crashed because it wasn't wrapped into the unit cell (which was defined as zeros, causing a bad error). This change fixes that.

Reply:
no, I mean leave out the ctime/user/unique_id fields, keeping the cell & position changes